src/rsz/src/Resizer.cc

```diff
@@ -321,7 +321,7 @@ Resizer::bufferConnectedToPorts(Instance *buffer)
   Net *out_net = db_network_->net(out_pin);
```

Comment:
bufferConnectedToPorts is no longer an appropriate function name.
bufferBetweenPorts is more appropriate.

The comment in the caller needs to be updated in the same way.